packages/venia-concept/src/components/MyAccountMenuPage/UserInformation/userInformation.js

```diff
@@ -1,44 +1,44 @@
-import React, { Component } from 'react';
-import PropTypes from 'prop-types';
-import classify from 'src/classify';
+import React from 'react';
+import { shape, string } from 'prop-types';
+
+import { mergeClasses } from 'src/classify';
 import Icon from 'src/components/Icon';
 import { User as UserIcon } from 'react-feather';
 import defaultClasses from './userInformation.css';
 
-class UserInformation extends Component {
-    static propTypes = {
-        classes: PropTypes.shape({
-            root: PropTypes.string,
-            userInformationContainer: PropTypes.string,
-            userInformationSecondary: PropTypes.string,
-            iconContainer: PropTypes.string
-        }),
-        user: PropTypes.shape({
-            email: PropTypes.string,
-            firstname: PropTypes.string,
-            lastname: PropTypes.string,
-            fullname: PropTypes.string
-        })
-    };
-
-    render() {
-        const { user, classes } = this.props;
-        const { fullname, email } = user || {};
-
-        const display = fullname.trim() || 'Loading...';
+const UserInformation = props => {
+    const { user } = props;
```

Comment:
```suggestion
    const { user  = {} } = props;
```

I think this reads better as defaulting instead of the below line.